packages/react-components/theme-designer/src/components/Sidebar/Sidebar.tsx

```diff
@@ -30,6 +30,18 @@ export const Sidebar: React.FC<SidebarProps> = props => {
 
   const sidebarId = useId();
 
+  // const [tab, setTab] = React.useState<TabValue>('use');
+  // const handleTabChange = (event: SelectTabEvent, data: SelectTabData) => {
+  //   // this is outdated, but i'm going to leave it until arman gives me more clarity on what the sidebar will look like
+  //   if (data.value === 'edit') {
+  //     dispatchAppState({ type: 'Custom', customAttributes: formState, overrides: {} });
+  //   } else if (data.value === 'use') {
+  //     setTheme('Custom');
+  //   }
+  //   setTab(data.value);
+  // };
+
+  // const [theme, setTheme] = React.useState<string>('Web');
```

Comment:
This is the only change that was not part of any cherry-picks. This was erroring out in CI since these are unused code, it's largely inconsequential for the web-components-v3 branch

Reply:
What's the reason `theme-designer` changes are included in the first place? Somewhat surprised to see them here.

Reply:
The old version of theme-designer had a dependency on `@fluent-blocks/colors` which wasn't compatible with node 18 therefore I cherry-picked theme-designer related changes into this PR up to a point where that dependency was removed